src/main/java/com/dev/cinema/model/dto/UserRequestDto.java

```diff
@@ -1,8 +1,24 @@
 package com.dev.cinema.model.dto;
 
+import com.dev.cinema.validation.Email;
+import com.dev.cinema.validation.FieldsValueMatch;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@FieldsValueMatch.List({
+        @FieldsValueMatch(
+                field = "password",
+                fieldMatch = "passwordRepeat",
+                message = "Passwords do not match!"
+        )
+})
```

Comment:
```suggestion
@FieldsValueMatch(
                field = "password",
                fieldMatch = "passwordRepeat",
                message = "Passwords do not match!"
)
```

what if you try this and try removing `@List` from inside `@FieldsValueMatch`?

Reply:
done